ibis/backends/tests/test_client.py

```diff
@@ -1750,3 +1751,27 @@ def test_cross_database_join(con_create_database, monkeypatch):
     con.drop_table(left_table)
     con.drop_table(right_table, database=dbname)
     con.drop_database(dbname)
+
+
+@pytest.mark.notyet(
+    ["druid"], raises=NotImplementedError, reason="doesn't support create_table"
+)
+@pytest.mark.notimpl(["clickhouse"], reason="create table isn't implemented")
+@pytest.mark.notyet(["pandas", "dask", "polars"], reason="Doesn't support insert")
+@pytest.mark.notyet(["exasol"], reason="Backend does not support raw_sql")
+@pytest.mark.notimpl(
+    ["impala", "pyspark", "trino"], reason="Default constraints are not supported"
+)
+def test_insert_into_table_missing_columns(con):
+    table_name = gen_name("table")
+    ct_sql = f"CREATE TABLE {table_name} (a INT DEFAULT 1, b VARCHAR(10));"
```

Comment:
Definitely need to aggressively quote every identifier here.

Reply:
Good call, thank you! After quoting, I got past that portion of the error, but ran into a different error from VARCHAR(10) getting translated to a CLOB type, which didn't play nicely with my assertion. I just adjusted to test with two int columns this time around. All seems to be working on the Oracle side now.